src/charm.py

```diff
@@ -32,7 +32,7 @@ class MySQLCharm(CharmBase):
     def __init__(self, *args):
         super().__init__(*args)
         self._stored.set_default(
-            mysql_setup={"MYSQL_ROOT_PASSWORD": False},
+            mysql_setup={"mysql_root_password": False},
```

Comment:
Any reason not to
```python
mysql_root_password = "mysql_root_password"
```
and use `mysql_root_password` as a (refactorable, safer) variable instead?